datanode/integration/oracles_test.go

```diff
@@ -28,6 +28,17 @@ func TestOracles(t *testing.T) {
 		})
 	}
 
+	queries = map[string]string{
+		//"OracleDataSourceExternalEthereum": `{ oracleSpecsConnection { edges { node { dataSourceSpec { spec { id createdAt updatedAt status data { sourceType { ... on DataSourceDefinitionExternal { sourceType { ... on EthCallSpec { address abi method args trigger requiredConfirmations filters } } } } } } } } } } }`,
```

Comment:
@AnNiran Is this dead code a reminder of something still to do here?

Reply:
Yes, this is an integration test for EthOracle i need to fix

Reply:
```suggestion
```